equinix/resource_fabric_cloud_router.go

```diff
@@ -231,14 +231,14 @@ func resourceCloudRouterDelete(ctx context.Context, d *schema.ResourceData, meta
 		return diag.FromErr(equinix_errors.FormatFabricError(err))
 	}
 
-	err = waitUntilCloudRouterDeprovisioned(d.Id(), meta, ctx)
+	err = WaitUntilCloudRouterDeprovisioned(d.Id(), meta, ctx)
 	if err != nil {
 		return diag.FromErr(fmt.Errorf("API call failed while waiting for resource deletion. Error %v", err))
 	}
 	return diags
 }
 
-func waitUntilCloudRouterDeprovisioned(uuid string, meta interface{}, ctx context.Context) error {
+func WaitUntilCloudRouterDeprovisioned(uuid string, meta interface{}, ctx context.Context) error {
```

Comment:
This is another function that was referenced in acceptance tests and customer-facing code.  Again, the aim of this PR is to get acceptance tests isolated with minimal logic changes, so I've opted to make this function public.  When this resource is migrated to a separate package there will be an opportunity to revisit this choice.